common/src/main/java/com/wynntils/wynn/model/CharacterSelectionManager.java

```diff
@@ -49,11 +50,13 @@ public CharacterSelectionManager() {
 
     @SubscribeEvent
     public void onScreenOpened(ScreenOpenedEvent event) {
-        if (event.getScreen() instanceof CharacterSelectorScreen characterSelectorScreen) {
```

Comment:
Here is where we do the instance check. I tried turning this into a utility function, but it was kind of pointless. In the end you still want to cast it to a specific subclass of screen, and that could not be done by a method. At least not without a bunch of generics black magic. Unless we get more instances, I suggest we skip doing a util function, ok?

Reply:
Okay.